app/views/layouts/application.html.slim

```diff
@@ -41,3 +41,4 @@ html
 
     javascript:
       var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
+      var SALESFORCE_BASE_URL = "#{{current_user&.admin ? ENV['SALESFORCE_INSTANCE_URL'] : ENV['COMMUNITY_LOGIN_URL']}}"
```

Comment:
is this an accidental duplication of this line? or is there a difference

Reply:
This was an accident, removing it now